lib/service/storage/virtualIndex.ts

```diff
@@ -0,0 +1,157 @@
+import Crypto from "crypto";
+
+import { Service } from "../service";
+
+export class VirtualIndex extends Service {
+  public static createEvent = "core:storage:virtualindex:cache:create";
```

Comment:
We use kebab case in events name (apart from public events because we don't want breaking)
```suggestion
  public static createEvent = "core:storage:virtual-index:cache:create";
```

Reply:
done